beacon/sync/src/test/java/tech/pegasys/teku/beacon/sync/forward/multipeer/chains/ThrottlingSyncSourceTest.java

```diff
@@ -129,6 +151,26 @@ void shouldDelayRequestIfBlobsSidecarsLimitAlreadyExceeded() {
             .requestBlobsSidecarsByRange(UInt64.valueOf(100), count, blobsSidecarsListener));
   }
 
+  @Test
+  void shouldDelayRequestIfBlobSidecarsLimitAlreadyExceeded() {
+    final UInt64 count = UInt64.valueOf(MAX_BLOB_SIDECARS_PER_MINUTE);
+    ignoreFuture(source.requestBlobSidecarsByRange(UInt64.ZERO, count, blobSidecarsListener));
+    ignoreFuture(
+        source.requestBlobSidecarsByRange(UInt64.valueOf(100), count, blobSidecarsListener));
+
+    // Both requests happen immediately
```

Comment:
I have a gut feeling that comment "// Both requests happen immediately" was copied from first "shouldRequestImmediately" test and now is copied to all other tests including throttled. Someone should stop this :)
So I think it's correct only in immediate test cases

Reply:
Done